metricbeat/README.md

```diff
@@ -141,6 +212,9 @@ helm uninstall --namespace=kube-system logzio-k8s-metrics
 
 
 ## Change log
+ - **0.0.3**:
+    - Upgrade to Metricbeat version 7.9.1.
```

Comment:
Maybe add a `breaking changes` note for the Metricbeat update?

Reply:
Do we even support this version in our internal dashboards?

Reply:
We do. Tested regular flow to all k8s dashboards with Daniel. @yyyogev

Reply:
Splendid!